README.md

```diff
@@ -109,3 +109,9 @@ python -c "import ssl; print ssl.OPENSSL_VERSION"
 ```
 
 You'll need a version with OpenSSL version 1.0.0 or later.
+
+### Hostname doesn't match
+
+If you get an ssl.CertificateError complaining about hostname match, your installed packages does not meet version [requirements](requirements.txt). 
+Specificly check ipaddress and urllib3 package versions to make sure they met requiurments in [requirements.txt](requirements.txt) file.
```

Comment:
Typos: requiurments  -> requirements.  Specificly -> Specifically.

Also, I'd add backquotes to the packages, like `ipaddress`.